chrome/content/otr.js

```diff
@@ -354,6 +354,31 @@ var otr = {
       otr.writeFingerprints();
   },
 
+  addFingerprint: function(context, hex) {
+    let fingerprint = new libOTR.hash_t();
+    if (hex.length != 40) throw new Error("Invalid fingerprint value.");
+    let bytes = hex.match(/.{1,2}/g);
+    for (let i = 0; i < 20; i++) {
+      fingerprint[i] = parseInt(bytes[i], 16);
+    }
+    return libOTR.otrl_context_find_fingerprint(context._context, fingerprint, 1, null);
+  },
+
+  hasContactFingerprint: function(args) {
```

Comment:
In light of the above, this should probably be `getFingerprintsForRecipient` and return an `Array`, and then that the length is `0` for has.